engineering/workflow.md

```diff
@@ -54,31 +55,153 @@ The team conducts a Sprint Planning meeting for 60 minutes at the beginning of e
 - Not have so many items that a team member gets overwhelmed
 - Under-estimate our team's total capacity, to provide room for unexpected work (e.g., support work)
-- Have a team member assigned to each item on the board
+- Have at least one team member assigned to each item on the board
 :::
 
-The Sprint Board is broken down into these columns:
+The Sprint Board is broken down into different columns that represent the teams delivery workflow. The team owns the design of this workflow and should change the workflow process that best suits their way of working to optimize for sustainable delivery.
+
+The current queues of work represented by the board are:
+
+- {guilabel} `Backlog` represents items that haven't been started and not in Sprint.
+- {guilabel} `Refined` represents a list of items that have been refined and are ready to be worked on. The items in this queue are not committed items.
+- {guilabel} `Committed` items haven't been started and are expected to be completed by end of the Sprint. These items are identified during the Iteration Planning activity.
+- {guilabel} `In Progress` represents a queue of items being *actively* worked on. Each item should have at least one owner.
+- {guilabel} `In Review/Blocked` contains items that need to be review before being marked as done or that cannot be completed without additional actions/support. A long list of items in this queue indicate deficiencies in the team's Refinement activity.
+- {guilabel} `Done` contains the list of items that are complete! These should be celebrated archived in the next Sprint Planning meeting.
+
+
+(meetings:retrospective)=
+### The Retrospective Meeting
+
+At the end of each sprint, the team holds a retrospective meeting to reflect and identify actions to improve the team's ways of working and delivery process. The retrospective is the process through which the team achieves continuous improvement for **all** their other processes.
+
+#### The Duration of the Retrospective
+
+The retrospective is one hour long and is usually held on the last day of each sprint.
+It is held at a time to maximise attendance from the engineering team.
+
+On rare occasions, when the team experiences duress or unpredictable and disruptive events, they may choose to have a specific retrospective to learn from those events.
+
+% TODO: Move the meeting into the team calendar for visibility: https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com
+
+#### The Roster for Facilitating Retrospectives
+
+There is a [Google Sheet](https://docs.google.com/spreadsheets/d/1s0ZSAxwFzJ-_WgDkZeicfuUadSFEjkjS6ZcNk4N7Mmg/edit?usp=sharing) in the shared team drive that determines who will be facilitating each retrospective meeting (as well as [sprint planning](meetings:sprint-planning) and backlog refinement meetings).
+Members of the engineering team are expected to self-nominate for this role because it is their improvement process.
```

Comment:
Emphasise ownership!
```suggestion
Members of the engineering team are expected to self-nominate for this role because it is _their_ improvement process.
```